docs/criufeature.md

```diff
@@ -0,0 +1,93 @@
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# CRIU feature
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** CRIU support is a technical preview feature that is intended for container environments. All APIs and options are subject to change. CRIU support requires the version of CRIU that is a package in Semeru UBI container images.
+
+In Eclipse OpenJ9&trade;, Checkpoint Restore in User Space (CRIU) feature is available as an API that can be used to stop the VM at a particular checkpoint, save the state, and then enable the VM to resume from the point where it was stopped. The time between the VM start and checkpoint is referred to as the checkpoint phase. The checkpointed state is written to one or more image files with the memory content available during the checkpoint phase, such as memory pages, methods, file systems, processes, network connections, and security components. You can restore the checkpoint image from these files in as many environments as possible with improved start-up time and hence with better VM performance.
+
+For more information about the CRIU feature and OpenJ9 CRIU support, see the following blog posts:
+
+- [Fast JVM startup with OpenJ9 CRIU Support](https://blog.openj9.org/2022/09/26/fast-jvm-startup-with-openj9-criu-support/)
+- [Getting started with OpenJ9 CRIU Support](https://blog.openj9.org/2022/09/26/getting-started-with-openj9-criu-support/)
+- [OpenJ9 CRIU Support: A look under the hood](https://blog.openj9.org/2022/10/14/openj9-criu-support-a-look-under-the-hood/)
+- [OpenJ9 CRIU Support: A look under the hood (part II)](https://blog.openj9.org/2022/10/14/openj9-criu-support-a-look-under-the-hood-part-ii/)
+
+## Using the CRIU feature
+
+CRIU feature is not enabled by default, you must explicitly invoke it. You can invoke the feature with the `-XX:+EnableCRIUSupport` option. When thus enabled, the VM starts with support for checkpoint and restore. The VM will launch the JIT, GC, Security, and other internal components that make checkpoint and restore feasible.
```

Comment:
Can we just stick with "enabling". And we shouldn't say you can enable using the option because that suggests you can enable it in other ways. I suggest something like "CRIU support is not enabled by default; you must enable it by specifying the `-XX:+EnableCRIUSupport` command-line option when you start your application.
The 2nd and 3rd sentences are a given after you've enabled support, so don't need to be said again. And if you make the first mention of the option a link to the relevant topic, you don't need the last line in this section either.